README.md

```diff
@@ -6,54 +6,103 @@
 2. Make reading code easier by providing consistent code style.
 3. Make writing code faster by leveraging auto fix wherever possible.
 
-## Usage
+## Getting started
 
-1. `yarn add eslint eslint-config-molindo --dev`
-2. Setup your project config in `.eslintrc.js`:
+1. `yarn add eslint eslint-config-molindo prettier --dev`
+2. Setup your project config in `eslint.config.mjs`:
 
 ```js
-module.exports = {
-  // Add configs based on your needs
-  extends: [
-    'molindo/typescript', // Or `molindo/javascript`
-    
-    'molindo/react', // Optional
-    'molindo/css-modules', // Optional
-    'molindo/tailwind', // Optional
-    'molindo/jest', // Optional
-    'molindo/cypress' // Optional
-  ]
-}
+// @ts-check
+
+import {getPresets} from 'eslint-config-molindo';
+
+export default [
+  ...(await getPresets(
+    // Base config
+    'typescript', // or 'javascript'
+
+    // Optional extensions
+    'react',
+    'cssModules',
+    'tailwind',
+    'jest',
+    'cypress',
+    'vitest'
+  )),
```

Comment:
ESLint 9 requires to import plugins and presets. To avoiding having to declare presets both as an import as well as an item within this array, I've added a (type-checked) function that imports them by name.

An important aspect of this is that individual presets only execute when they are loaded (since the `typescript` preset depends on the `typescript` dependency, as well as the `tailwind` preset depends on the `tailwindcss` dependency).